### PR TITLE
[RSPECT MODEL] Booking model rspec

### PR DESCRIPTION
**Pull request by huytn-1183**
## Related Tickets
- [#43126](https://edu-redmine.sun-asterisk.vn/issues/43126)
- [#43127](https://edu-redmine.sun-asterisk.vn/issues/43127)

## WHAT (optional)
- create right user relationship
- create right default status
- create details in reverse chronologically
- pagination invalid page
- status filter by invalid string value

## HOW
- Rspect model

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by huytn-1183** [comment]
ready

**Comment by vanvtt-0952** [comment]
commented
Conflict code nè cu

**Comment by huytn-1183** [comment]
ready

File: spec/factories.rb [hunk 1]
```diff
@@ -20,3 +11,13 @@
     user { create(:user) }
   end
 end
+
+FactoryBot.define do
```
Comment (vanvtt-0952) [inline spec/factories.rb:15]: tường là sẽ tách ra các file factories theo model á bé

**Comment by vanvtt-0952** [comment]
merged
Chị cso comment
em fix pull kế tiếp nhé

**Pull request merged**